src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -140,9 +140,11 @@ private Content generateHtml(SlingHttpServletRequest request, Resource resource)
             ob = ob.templateDir(dependencyProvider.getTemplateDir());
         }
 
+        long start = System.currentTimeMillis();
         c.html = getDependencyProvider().getAsciidoctor().convert(
                 c.asciidoc,
                 ob.get());
+        System.out.println("Rendering finished in " + (System.currentTimeMillis() - start) + " ms.");
```

Comment:
Lets use the `log` variable here, instead of `System.out`.

Reply:
Boooooo! System.out FOREVER! :)